lib/job/arFindingAidJob.class.php

```diff
@@ -203,22 +327,59 @@ public static function getStatus($id)
     ';
 
     $ret = QubitPdo::fetchOne($sql, array(get_class(), $id));
+
     return $ret ? (int)$ret->statusId : null;
   }
 
   public static function getStatusString($id)
   {
+    $i18n = sfContext::getInstance()->i18n;
+
     switch (self::getStatus($id))
     {
       case QubitTerm::JOB_STATUS_COMPLETED_ID:
-        return 'completed';
+        return $i18n->__('completed');
+
       case QubitTerm::JOB_STATUS_IN_PROGRESS_ID:
-        return 'generating';
+        return $i18n->__('in progress');
+
       case QubitTerm::JOB_STATUS_ERROR_ID:
-        return 'error';
+        return $i18n->__('error');
+
       default:
-        return 'unknown';
+        return $i18n->__('unknown');
+    }
+  }
```

Comment:
:+1: 

Also if you plan to use the above code in QubitInformationObject, use isset() for the if statement.